assume/strategies/flexable_storage.py

```diff
@@ -0,0 +1,104 @@
+import numpy as np
+import pandas as pd
+
+from assume.strategies.base_strategy import BaseStrategy
+from assume.units.storage_unit import StorageUnit
+
+
+class flexableEOMStorage(BaseStrategy):
+    def __init__(self):
+        super().__init__()
+
+        self.foresight = pd.Timedelta("12h")
+        self.current_time = None
+
+    def calculate_bids(
+        self,
+        unit: StorageUnit = None,
+        operational_window: dict = None,
+    ):
+        """
+        Takes information from a unit that the unit operator manages and
+        defines how it is dispatched to the market
+
+        Return: volume, price
+        Strategy analogue to flexABLE
+
+        """
+        if operational_window is not None:
+            # =============================================================================
+            # Storage Unit is either charging, discharging, or off
+            # =============================================================================
+            self.current_time = operational_window["window"]["start"]
```

Comment:
That is not correct.
The current time is not equal to the start of the current bidding window for which the bid calculation was requested.
I think you can fix this by copying the changes from the powerplant, in which this has een changed

Reply:
This is then also wrong in flexable.py. I'll try to correct both then